Sources/Sentry/SentryUIViewControllerSwizziling.m

```diff
@@ -31,27 +41,38 @@ + (void)startWithOptions:(SentryOptions *)options
  * Swizzle the some init methods of the view controller,
  * so we can swizzle user view controller subclass on demand.
  */
-+ (void)swizzleViewControllerInits
++ (void)swizzleViewControllers
```

Comment:
I like this idea. I guess being executed async, the performance will not be an issue.
This way we don't mess with the initialization, which probably is the cause of the iOS 15 crash.